test/e2e/instascale_machineset_test.go

```diff
@@ -0,0 +1,55 @@
+package e2e
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+	mcadv1beta1 "github.com/project-codeflare/multi-cluster-app-dispatcher/pkg/apis/controller/v1beta1"
+
+	. "github.com/project-codeflare/codeflare-operator/test/support"
+)
+
+func TestInstascaleMachineSet(t *testing.T) {
+	test := With(t)
+	test.T().Parallel()
+
+	// skip test if not using machine sets
+	clusterType := GetClusterType(test)
+	if clusterType != OcpCluster {
+		test.T().Skipf("Skipping test as not running on an OCP cluster, resolved cluster type: %s", clusterType)
+	}
+
+	namespace := test.NewTestNamespace()
+
+	// Test configuration
+	cm := CreateConfigMap(test, namespace.Name, map[string][]byte{
+		// pip requirements
+		"requirements.txt": ReadFile(test, "mnist_pip_requirements.txt"),
+		// MNIST training script
+		"mnist.py": ReadFile(test, "mnist.py"),
+	})
+
+	// look for machine set with aw name - expect to find it
+	test.Expect(GetMachineSets(test)).Should(ContainElement(WithTransform(MachineSetId, Equal("test-instascale"))))
```

Comment:
Could `test-instascale` be declared once? Maybe `createInstaScaleJobAppWrapper` could be turned into a factory method instead of creating the AppWrapper in the cluster, and `aw.Name` be used consistently.

Reply:
Like this? https://github.com/project-codeflare/codeflare-operator/pull/322/commits/7e38eb26c755d9171389726fc6484d12efc02f9c

Reply:
yes exactly. Just renaming `createInstaScaleJobAppWrapper` to `instaScaleJobAppWrapper` maybe?